inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderListPanel.html

```diff
@@ -18,18 +18,21 @@
 -->
 <html xmlns:wicket="http://wicket.apache.org">
 <body>
-  <wicket:panel>
-    <div class="flex-content panel panel-default panel-flex">
-      <div class="panel-heading">
-        <h3 class="panel-title"><wicket:message key="recommenders"/></h3>
-      </div>
-      <div class="panel-body fit-child-snug">
-        <select class="form-control" wicket:id="recommenders"></select>
-      </div>
-      <div class="panel-footer text-right">
-        <input wicket:id="create" type="button" class="btn btn-primary" wicket:message="value:create"/>
-      </div>            
-    </div>
-  </wicket:panel>
+	<wicket:panel>
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.